src/electron/platform/android/fetch-scan-results.ts

```diff
@@ -3,6 +3,8 @@
 import axios from 'axios';
 import { ScanResults } from './scan-results';
 
+export type FetchScanResults = (port: number) => Promise<ScanResults>;
```

Comment:
rename it to `FetchScanResultsType` or something, maybe?

Reply:
Renamed!